frontend/src/app/bounty/create/page.tsx

```diff
@@ -142,12 +145,32 @@ const CreateBountyPage: FC = () => {
                         onChange={(e) => setDeadline(e)}
                     />
 
-                    <FileDrop
-                        onDrop={(files) => readFile(files[0])}
-                        accept={{ "application/octet-stream": [".tar.xz"] }}
-                    >
-                        <FileDropText filename={filename} />
-                    </FileDrop>
+                    <Tabs defaultValue="file">
+                        <Tabs.List>
+                            <Tabs.Tab value="file">Upload</Tabs.Tab>
+                            <Tabs.Tab value="path">Built-in</Tabs.Tab>
+                        </Tabs.List>
+
+                        <Tabs.Panel value="file">
+                            <FileDrop
+                                onDrop={(files) => readFile(files[0])}
+                                accept={{
+                                    "application/octet-stream": [".tar.xz"],
+                                }}
+                            >
+                                <FileDropText filename={filename} />
+                            </FileDrop>
+                        </Tabs.Panel>
+
+                        <Tabs.Panel value="path">
+                            <TextInput
+                                size="lg"
+                                value={appPath ?? ""}
+                                placeholder="/bounties/*.tar.xz"
```

Comment:
Small suggestion to change the current placeholder for: "/bounties/a_builtin_bounty_file.tar.xz" or something like that, but it is really a suggestion with no strong argument.